2015-Fall/adrianbrink/SmashTag/SmashTag/TweetTableViewCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  TweetTableViewCell.swift
+//  SmashTag
+//
+//  Created by aevitas on 15/10/15.
+//  Copyright © 2015 adrian. All rights reserved.
+//
+
+import UIKit
+
+class TweetTableViewCell: UITableViewCell {
+    
+    var tweet: Tweet? {
+        didSet {
+            updateUI()
+        }
+    }
+
+    @IBOutlet weak var tweetProfileImageView: UIImageView!
+    @IBOutlet weak var tweetScreenNameLabel: UILabel!
+    @IBOutlet weak var tweetTextLabel: UILabel!
+    
+    func updateUI() {
+        tweetTextLabel?.attributedText = nil
+        tweetScreenNameLabel?.text = nil
```

Comment:
Your app crashed for me because you named this tweetScreenName in the story board. To fix this: storyboard > select prototype cell > open utilities pane>tap "Show the connections connections inspector" (right most tab in the pane) > delete the erroneous outlet > reconnect outlet

or

Rename tweetScreenNameLabel to tweetScreenName